docker-compose.env

```diff
@@ -35,6 +35,7 @@ MNEMONIC='abandon abandon abandon abandon abandon abandon abandon abandon abando
 
 # used by deployer
 HARDHAT=true
+L2_CROSS_DOMAIN_MESSENGER_ADDRESS=0x4200000000000000000000000000000000000007
```

Comment:
Is this meant to be this? https://github.com/ethereum-optimism/go-ethereum/pull/109/files#diff-7aa83061ca1c69a8ab9203da49a7581bad6e7cc80ecb96582e7fb09c0b8f6534R815

Reply:
Note that the name of the env var is different